[2025-02-07T11:12:14.319+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T11:12:14.339+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:21:00+00:00 [queued]>
[2025-02-07T11:12:14.351+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:21:00+00:00 [queued]>
[2025-02-07T11:12:14.351+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-07T11:12:14.366+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_physical_store_data> on 2025-02-06 00:21:00+00:00
[2025-02-07T11:12:14.377+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=146) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T11:12:14.380+0000] {standard_task_runner.py:72} INFO - Started process 150 to run task
[2025-02-07T11:12:14.381+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_physical_store_data', 'scheduled__2025-02-06T00:21:00+00:00', '--job-id', '590', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpix0vo_9z']
[2025-02-07T11:12:14.383+0000] {standard_task_runner.py:105} INFO - Job 590: Subtask extract_physical_store_data
[2025-02-07T11:12:14.457+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:21:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-07T11:12:14.556+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_physical_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:21:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:21:00+00:00'
[2025-02-07T11:12:14.557+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T11:12:14.557+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T11:12:14.557+0000] {logging_mixin.py:190} INFO - Current task name:extract_physical_store_data state:running start_date:2025-02-07 11:12:14.340054+00:00
[2025-02-07T11:12:14.558+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-07T11:12:14.558+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T11:12:14.576+0000] {base.py:84} INFO - Retrieving connection 'physical_store_conn'
[2025-02-07T11:12:14.582+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM physical_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-07T11:12:14.600+0000] {sql.py:562} INFO - Rows affected: 9342
[2025-02-07T11:12:14.750+0000] {etl.py:203} INFO - Extracted 9342 new rows from db_physical_store.
[2025-02-07T11:12:14.754+0000] {python.py:240} INFO - Done. Returned value was: /tmp/physical_data.json
[2025-02-07T11:12:14.774+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T11:12:14.775+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_physical_store_data, run_id=scheduled__2025-02-06T00:21:00+00:00, execution_date=20250206T002100, start_date=20250207T111214, end_date=20250207T111214
[2025-02-07T11:12:14.785+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T11:12:14.785+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T11:12:14.785+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-07 11:12:08.972922+00:00
[2025-02-07T11:12:14.785+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-07T11:12:14.836+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T11:12:14.857+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-07T11:12:14.858+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
